pom.xml

```diff
@@ -55,6 +55,44 @@
         <test.on.emualator>false</test.on.emualator>
     </properties>
 
+    <repositories>
+        <repository>
+            <id>spring-snapshots</id>
```

Comment:
I think spring data documentdb release could wait till spring boot 2.0 official released right?

Reply:
given it's on in spring initializr